### ISSUE TITLE
Hovering over template reference throws Error

### ISSUE THREAD

**Issue by frederikschubert** [angular/vscode-ng-language-service#121]
Using version 0.1.4 of the language service. Hovering over the `chart` reference throws the following error.

## Template
```html
<div #chart>
</div>
```

## Error
```
[Error - 08:58:34] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'identifier' of null
  Code: -32603 
[Error - 08:58:35] SERVER ERROR: Cannot read property 'identifier' of null
TypeError: Cannot read property 'identifier' of null
    at tokenReference (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/node_modules/@angular/language-service/bundles/language-service.umd.js:18980:14)
    at Object.visitReference (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/node_modules/@angular/language-service/bundles/language-service.umd.js:50335:62)
    at ReferenceAst.visit (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/node_modules/@angular/language-service/bundles/language-service.umd.js:17198:24)
    at locateSymbol (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/node_modules/@angular/language-service/bundles/language-service.umd.js:50314:22)
    at getHover (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/node_modules/@angular/language-service/bundles/language-service.umd.js:50589:18)
    at LanguageServiceImpl.getHoverAt (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/node_modules/@angular/language-service/bundles/language-service.umd.js:50662:20)
    at /Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/editorServices.js:2705:68
    at time (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/editorServices.js:2687:22)
    at Object.getHoverAt (/Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/editorServices.js:2705:20)
    at /Users/frederik/.vscode/extensions/Angular.ng-template-0.1.4/server/server.js:171:30
```

**Comment by chuckjaz** [comment]
Tracking in: https://github.com/angular/angular/issues/17972

**Comment by erathostene** [comment]
Somehow related?
Hovering over "class" (or "ngClass" extended by @angular/flex-layout : "ngClass.gt-sm") throw errors (and open console to display them)
### Template
```` html
<div class="logo">...</div>
````
### Error
````
Search path: d:/informatique/PROJETS/Noos/matete-client/src/app/sidenav
Config file name: d:/informatique/PROJETS/Noos/matete-client/tsconfig.json
Angular Language Service: 4.3.0-beta.0-1fc0d05
TypeScript: 2.3.4 // NB : Not my actual typescript version (2.4.0)
getTemplateRefrences: 2104ms
Add recursive watcher for: d:/informatique/PROJETS/Noos/matete-client
Opened configuration file d:/informatique/PROJETS/Noos/matete-client/tsconfig.json
getTemplateRefrences: 1ms
[Error - 11:35:06] SERVER ERROR: Cannot read property 'flags' of undefined
TypeError: Cannot read property 'flags' of undefined
    at SymbolWrapper.members (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:48129:29)
    at findInputBinding (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:50394:40)
    at Object.visitDirectiveProperty (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:50367:32)
    at BoundDirectivePropertyAst.visit (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:17332:24)
    at locateSymbol (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:50314:22)
    at getHover (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:50589:18)
    at LanguageServiceImpl.getHoverAt (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\node_modules\@angular\language-service\bundles\language-service.umd.js:50662:20)
    at C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\editorServices.js:2705:68
    at time (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\editorServices.js:2687:22)
    at Object.getHoverAt (C:\Users\antoine\.vscode\extensions\Angular.ng-template-0.1.4\server\editorServices.js:2705:20)
[Error - 11:35:06] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'flags' of undefined
  Code: -32603 
getDiagnostics: 62ms
````

**Comment by chuckjaz** [comment]
Closed with 0.1.6
